<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableRecords</class>
 <widget class="QWidget" name="TableRecords">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>354</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TableRecords</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="label_Icon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/icons/icons/glyphicons_072_bookmark.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Title">
       <property name="text">
        <string>发货记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Filter">
       <property name="text">
        <string>筛选 (&amp;F)</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/glyphicons_027_search.png</normaloff>:/icons/icons/glyphicons_027_search.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+F</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView_Records"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,1,0,0,0">
     <item>
      <widget class="QPushButton" name="pushButton_LastPage">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/glyphicons_216_circle_arrow_left.png</normaloff>:/icons/icons/glyphicons_216_circle_arrow_left.png</iconset>
       </property>
       <property name="shortcut">
        <string>PgUp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Page">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_NextPage">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/glyphicons_217_circle_arrow_right.png</normaloff>:/icons/icons/glyphicons_217_circle_arrow_right.png</iconset>
       </property>
       <property name="shortcut">
        <string>PgDown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Reload">
       <property name="text">
        <string>刷新 (F5)</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/glyphicons_081_refresh.png</normaloff>:/icons/icons/glyphicons_081_refresh.png</iconset>
       </property>
       <property name="shortcut">
        <string>F5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Delete">
       <property name="text">
        <string>删除 (&amp;D)</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/glyphicons_191_circle_minus.png</normaloff>:/icons/icons/glyphicons_191_circle_minus.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Modify">
       <property name="text">
        <string>修改 (&amp;E)</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/glyphicons_193_circle_ok.png</normaloff>:/icons/icons/glyphicons_193_circle_ok.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+E</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_New">
       <property name="text">
        <string>新增 (&amp;N)</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/glyphicons_190_circle_plus.png</normaloff>:/icons/icons/glyphicons_190_circle_plus.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+N</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
